Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -588,6 +588,70 @@ kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o jsonpa
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o jsonpath='{.data.SecretKey}' | base64 --decode
 ```
 
+### Managing User S3 Credentials
+
+The default behavior of `CephObjectStoreUser` is to place the s3 credentials generated when an rgw user is created into a Kubernetes `Secret`.
```

Comment:
I'd like to make sure we are capitalizing S3 and RGW in our official docs. 

I also don't think we have a trend of putting resource kind names in code blocks. Referring to it as Kubernetes Secret seems appropriate to me. Similarly CephObjectStore user is sufficient in context.

```suggestion
The default behavior of `CephObjectStoreUser` is to place the S3 credentials generated when an RGW user is created into a Kubernetes Secret.
```